src/test/librbd/test_notify.py

```diff
@@ -100,9 +100,10 @@ def slave(ioctx):
 
     print("detected master")
 
-    print("rename")
-    RBD().rename(ioctx, CLONE_IMG_NAME, CLONE_IMG_RENAME);
-    assert(not image.is_exclusive_lock_owner())
+    with Image(ioctx, CLONE_IMG_NAME) as image:
+        print("rename")
+        RBD().rename(ioctx, CLONE_IMG_NAME, CLONE_IMG_RENAME);
+        assert(not image.is_exclusive_lock_owner())
```

Comment:
Just wandering if this assert is useful here. If the lock had been stolen it would have been stolen by the image context created by `rename` internally?

Reply:
If the lock was stolen, that indicates that the RPC message wasn't sent to the lock owner (plus the "master" waits for its lock to get stolen to know the test is over via the "slave" issuing a write).

Reply:
@dillaman not sure I understand. I mean that if "rename" had stolen the lock by some reason, the assert `not image.is_exclusive_lock_owner()` would have been still valid (the test would not fail at this point), because the lock is not stolen by this image context. Sure the test would fail later but I am talking only about usefulness of this assert. Am I missing something?

Reply:
Ah -- sorry, I see what you are saying. I was just trying to fix previous bug where ```image``` wasn't even initialized. I'll just drop it.